Makefile

```diff
@@ -13,8 +13,6 @@ counterfeiter := $(GOBIN)/counterfeiter
 mockgen := $(GOBIN)/mockgen
 IMAGE_REPO := quay.io/operator-framework/olm
 IMAGE_TAG ?= "dev"
-KUBE_DEPS := api apiextensions-apiserver apimachinery code-generator kube-aggregator kubernetes
-KUBE_RELEASE := release-1.11
```

Comment:
Did we want to just switch this to `release-1.12`?

Reply:
No, I couldn't find a way to have client-go and go modules work together without pinning to a specific release. So no more branch tracking, it's all in go.mod.